#!/usr/bin/env python
PACKAGE = "kinikun"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("p_gain", double_t, 0, "P gain", 1.0, 0.0, 100.0)
gen.add("i_gain", double_t, 0, "I gain", 0.0, 0.0, 10.0)
gen.add("d_gain", double_t, 0, "D gain", 0.0, 0.0, 100.0)
gen.add("target_angle", double_t, 0, "Target angle in degrees", 0.0, -45.0, 45.0)

exit(gen.generate(PACKAGE, "kinikun", "PID"))
